pkg/sql/logictest/testdata/logic_test/.gitattributes

```diff
@@ -0,0 +1 @@
+ranges diff
```

Comment:
It seems github doesn't respect this, but I made a comments-only change to the ranges file. So you can pull down the branch and see that you do indeed get a git-diff on this file now.  I don't feel strongly about this either way.